### ISSUE TITLE
DD reactivity incorrect

### ISSUE THREAD

**Issue by OceanNuclear** [Fusion-Power-Plant-Framework/bluemira#2888]
## Describe the bug/ programming error

The current [plasma_physics/reactions.py](https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/fb8a0513cbefa61b64435a3427544b108afa21eb/bluemira/plasma_physics/reactions.py#L370) assumes that the DD-reaction rate is calculated by 0.5 * (dd->3He,n) +  0.5 * (dd->tp). This is incorrect.

The reactivity is proportional to <σv>, which is an integral of some product of the target cross-section σ(v) with the velocity distribution, with respect to the velocity of the projectile. Therefore the reactivity of the reaction (dd->3He,n) is completely independent of the reactivity of (dd->tp). In order to get the total reactivity of DD reactions, one should add these two reactivities together, no branching ratio/fraction multipliers are required. Branching fraction is only relevant when trying to break down "total DD reactivity" back down to its constituent reactions' reactivity, i.e. (dd->3He,n) reactivity or (dd->tp) reactivity.

## Desired behaviour

reactivity of DD = reactivity of (dd->3He,n) + reactivity of (dd->tp).

**Comment by CoronelBuendia** [comment]
TLDR; still confused, not sure, need to have a coffee with a real plasma physicist.

I confess to never having really cared about reactivity per se, which I actually thought (wrongly, it seems?) by definition was <σv> - and not proportional to it.

I can't remember why I wanted this, but it could have been with a view to calculating the reaction rate (N.B. not equal to reactivity, and is: rate = n1 * n2 * <σv>), which in the case of D-D is frustrating because of n1*n2 having to be calculated as n1**2/2. 

Note that the issue states that the D-D reaction rate calculation is wrong, but we do not actually provide one.

I think we did briefly discuss that there is technically no such thing as D-D reactivity, seeing as that is not in fact a single channel, right? 

I'm not sure it really makes sense to have a "total" reactivity of two different reactions. You will find that the two D-D channels look pretty much identical in terms of reactivity on a log-log plot, which was another motivation for the average.
